test/test_cli.py

```diff
@@ -0,0 +1,102 @@
+"""Test for the CLI
+
+"""
+
+import os
+import sys
+from scripttest import TestFileEnvironment
+from nose.tools import assert_equals
+
+TEST_DIR = os.path.abspath('test/test-cli-output')
+
+DEVFILE_NAME = 'testfile'
+
+DEVFILE = os.path.join(TEST_DIR, DEVFILE_NAME)
+CONFIGFILE = 'testconfig.yaml'
+CONFIG_YAML = '''
+---
+
+printer:
+    type: file
+    devfile: {testfile}
+'''.format(
+    testfile=DEVFILE,
+)
+
+class TestCLI:
+    """ Contains setups, teardowns, and tests for CLI
+    """
+
+    @classmethod
```

Comment:
QuantifiedCode found this line. Is there any cause that you do not use a staticmethod?
Could you please change that to `@staticmethod` (and remove the `cls` in `(cls)`)?